index.js

```diff
@@ -18,8 +18,20 @@ function get(url, cb) {
 	});
 }
 
+function getCleanName(name){
+	name = name.split('@');
+	if(name.length === 2) {
+		name = '@' + encodeURIComponent(name[1]);
+	} else if(name.length === 1) {
+		name = encodeURIComponent(name[0]);
+	}
+	return name;
+}
```

Comment:
You're right. It's purpose is not clear, and tbh, I do not recall either. It is definitely required, though. I will re-discover why it was needed and add comments as appropriate.